src/rest.cpp

```diff
@@ -295,12 +295,12 @@ static bool rest_block(HTTPRequest* req,
 
 static bool rest_block_extended(const util::Ref& context, HTTPRequest* req, const std::string& strURIPart)
 {
-    return rest_block(req, strURIPart, true);
+    return rest_block(req, strURIPart, TxVerbosity::SHOW_DETAILS_AND_PREV_OUT);
```

Comment:
This could use a test

Reply:
I added a test in 37f2c4e9e09d586e1c7d38332dc7fe7864245b97